tests/firedrake/adjoint/test_burgers_newton.py

```diff
@@ -111,7 +115,9 @@ def J(ic, solve_type, timestep, steps, V):
 
     tape = get_working_tape()
     J = 0.0
-    for _ in tape.timestepper(range(steps)):
+    for j in tape.timestepper(iter(range(steps))):
```

Comment:
```suggestion
    for j in tape.timestepper(range(steps)):
```
don't think that is needed

Reply:
done